plugins/plugin.go

```diff
@@ -45,6 +46,7 @@ type Plugin struct {
 	MetricsHost       string
 	Log               *logrus.Entry
 	ConnectorFunc     func([]producers.MetricsMessage, *cli.Context) error
+	Client            *http.Client
```

Comment:
i would not export `http.Client` you are not going to use it from another package, if you do, something is not right and needs a refactor. Maybe you can explain why do you want to export it?